packages/typography/css/fonts.css

```diff
@@ -0,0 +1,42 @@
+@font-face {
+  font-display: swap;
+  font-family: 'Recoleta';
+  font-weight: 400;
+  src: url("https://s1.wp.com/i/fonts/recoleta/400.woff2") format("woff2"), url("https://s1.wp.com/i/fonts/recoleta/400.woff") format("woff"); }
+
+.wp-brand-font {
+  font-family: "Noto Serif", Georgia, "Times New Roman", Times, serif;
+  font-weight: 400; }
+
+[lang*=de] .wp-brand-font {
```

Comment:
This could be deduped with separating the selectors with a comma:
```css
[lang*=de] .wp-brand-font,
[lang*=en] .wp-brand-font,
[lang*=es] .wp-brand-font {
  font-family: "Recoleta", ...;
}
```
This is still basic CSS that doesn't require any SCSS processing.

Reply:
By the way, the reason why we limit Recoleta only to certain languages is that the font contains only Latin-1 (Western Europe) accented characters? And doesn't have the Latin-2 characters used by languages like Czech or Polish?

Reply:
For matching the `lang` HTML attribute, the `|=` attribute might be a better fit:
```
[lang|=de]
```
The [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors) mentions language codes as the typical use case for this operator:

> Represents elements with an attribute name of attr whose value can be exactly value or can begin with value immediately followed by a hyphen, - (U+002D). It is often used for language subcode matches.

Reply:
> By the way, the reason why we limit Recoleta only to certain languages is that the font contains only Latin-1 (Western Europe) accented characters? And doesn't have the Latin-2 characters used by languages like Czech or Polish?

Yep:

https://github.com/Automattic/wp-calypso/issues/42522

Reply:
In that case, we can also include Latin-2 languages like Czech, Polish, Slovak or Hungarian (`cs`, `pl`, `sk`, `hu`). The alphabet-type.com charset checker reports all required characters as present, and the only characters missing are some special punctuation marks. There are missing in, for example, Spanish, too.

It seems that the Vietnamese character set is quite an outlier here, with mere 57% characters present.